automated install/basic-install.sh

```diff
@@ -859,6 +863,10 @@ setStaticIPv4() {
     # Local, named variables
     local IFCFG_FILE
     local CONNECTION_NAME
+    # If a static interface is already configured, we are done.
+    if [[ -r /etc/sysconfig/network/ifcfg-${PIHOLE_INTERFACE} ]]; then
```

Comment:
Does this guarantee that the interface is set with a static IP address, or just that the interface exists and may still be assigned a dynamic address?

Reply:
OK, lets check the content too, to be sure.